javascript/src/index.js

```diff
@@ -6,6 +6,7 @@ import Auth0ProviderWithHistory from "main/components/Auth/Auth0ProviderWithHist
 import { BrowserRouter } from "react-router-dom";
 import * as serviceWorker from "./serviceWorker";
 import "bootstrap/dist/css/bootstrap.css";
+import "antd/dist/antd.css";
```

Comment:
Mainly asking for my own curiosity- why do we need this import here?

Reply:
This import includes the styles from the antd library into the app